shared/contracts/BaseTemplate.sol

```diff
@@ -179,6 +182,20 @@ contract BaseTemplate is APMNamehash, IsContract {
         _removePermissionFromTemplate(_acl, _tokenManager, _tokenManager.MINT_ROLE());
     }
 
+    /* SURVEY */
+
+    function _installSurveyApp(Kernel _dao, MiniMeToken _token, uint64 _minParticipationPct, uint64 _surveyTime) internal returns (Survey) {
+        Survey survey = Survey(_installNonDefaultApp(_dao, SURVEY_APP_ID));
+        survey.initialize(_token, _minParticipationPct, _surveyTime);
```

Comment:
Once #125 is merged, we should use the optimized version too :).

Reply:
Merging both.. will provide a third PR to update this :)